tests/test_entity_mixins.py

```diff
@@ -228,6 +228,15 @@ def test__poll_task_success(self):
                         entity_mixins._poll_task(gen_integer(), self.cfg),
                     )
 
+    def test__poll_task_timeout(self):
```

Comment:
```suggestion
    def test_poll_task_timeout(self):
```
Was the double underscore intentional?

Reply:
yes it is because we're testing the `_poll_task` function.